### PR TITLE
Tailwind integration

### PR DESCRIPTION
**Pull request by Sangeetha-Bheeman**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,16 +3,19 @@
...
+    "start": "npm-run-all -p serve serve:css",
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint --max-warnings 3",
     "pretty": "prettier --check '**/*'",
-    "test": "vue-cli-service test:unit && vue-cli-service test:e2e"
+    "test": "vue-cli-service test:unit && vue-cli-service test:e2e",
+    "serve:css": "postcss src/styles/tailwind.css -o src/styles/index.css"
```
Comment (KatyTaylor) [inline package.json:14]: If you look at the Readme, most of these commands are described in there, in various sections. Do you think you could add something to the Readme to describe what `yarn run start` and `yarn run serve:css` will do? Thanks

File: package.json [hunk 2]
```diff
@@ -23,6 +26,8 @@
     "https-proxy-agent": "^2.2.2",
     "jest-transform-graphql": "^2.1.0",
     "lodash-es": "^4.17.21",
+    "postcss": "^7",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
```
Comment (KatyTaylor) [inline package.json:30]: Just wondering why the format of this is different to all the other dependencies?

File: src/main.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ import store from './store'
 import BootstrapVue from 'bootstrap-vue'
 import { apolloProvider } from './graphql/client'
 import globalAlert from '@/mixins/globalAlert'
+import './styles/index.css'
```
Comment (KatyTaylor) [inline src/main.js:12]: Looks like this is a huge file - does this contain the tailwind framework? Is it correct to store it or should it get generated when we build the package?

File: tailwind.config.js [hunk 4]
```diff
@@ -0,0 +1,925 @@
+module.exports = {
+  content: ['./src/**/*.{html,vue}'],
+  // Used in the Labware component for slots, but colour is built dynamically so we need to make sure it's not purged
```
Comment (KatyTaylor) [inline tailwind.config.js:3]: I can't find the Labware component - is this your comment or copied over from STAN? Thanks

File: src/App.vue [hunk 5]
```diff
@@ -1,35 +1,44 @@
 <template>
-  <b-container id="app" fluid>
-    <br />
+  <div class="flex flex-col min-h-screen">
     <!-- TODO: move this into a header component -->
```
Comment (KatyTaylor) [inline src/App.vue:3]: I see you've created a 'Header' component. Do you still want this 'TODO' comment there? Or are you happy with what's in and out of the header component?

File: package.json (hunk 2 above)
Comment (seenanair) [reply package.json:30]: There is an earlier dependency in one of the existing packages in Traction that relies on an older version of PostCSS. So we need to use postcss7-compat along with tailwind, that is intended to use for this purpose.

File: src/main.js (hunk 3 above)
Comment (seenanair) [reply src/main.js:12]: Yes, you are right. The development build is usually large by design. According to tailwind documentation, to make the development environment more flexible, tailwind generates thousands of utility classes which most of which we probably don't use. But while buiding in production enviornment, Tailwind optimizes final build size by removing unused styles.

File: tailwind.config.js (hunk 4 above)
Comment (seenanair) [reply tailwind.config.js:3]: That's something that needs to be removed. will do it.

File: src/App.vue (hunk 5 above)
Comment (seenanair) [reply src/App.vue:3]: Removed

File: package.json (hunk 1 above)
Comment (seenanair) [reply package.json:14]: Done

**Review by KatyTaylor — DISMISSED** [review]
Hi @seenanair , thanks for answering all my comments. I've approved it now, although:
a) I can see you've got some merge conflicts now, and 
b) Steve said in stand up today that he was planning to have a look

**Pull request merged**